builder/builder.go

```diff
@@ -515,10 +515,10 @@ func (b *Builder) saveArtifactLocally(ctx context.Context, artifact domain.Artif
 				}
 			} else {
 				// Remove pre-existing dest dir.
-				err = os.RemoveAll(to)
-				if err != nil {
-					return errors.Wrapf(err, "rm -rf %s", to)
-				}
+				// err = os.RemoveAll(to)
```

Comment:
Do you suspect we'll revisit this commented out code, or can this else case be removed?

Reply:
It can be removed. I wanted to run all the tests, etc... wasn't quite ready for a review yet :D

Reply:
I  agree. Commented out code should be deleted, thats what source control is for :D

Reply:
hah, whoops, I think I just clicked on it without realising it was a draft :)

Reply:
Its not; I make my PR's from inside VSCode, and starting as a draft isn't an option :(